scripts/portconfig

```diff
@@ -90,6 +91,9 @@ class portconfig(object):
         lag_tables = self.db.get_table(PORT_CHANNEL_TABLE_NAME)
         lag_mbr_tables = self.db.get_table(PORT_CHANNEL_MBR_TABLE_NAME)
         if port in port_tables:
+            port_transceiver_info = self.state_db.get_all(self.state_db.STATE_DB, "TRANSCEIVER_INFO|{}".format(port))
+            if port_transceiver_info:
```

Comment:
what if port_transceiver_info is None?

Reply:
will use default value false, considered as a none RJ45 port,